packages/web3-profile/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@datamodels/web3-profile",
```

Comment:
Wouldn't it make more sense to call this `basic-profile`, as that would create a 1-1 mapping to the CIP?

Reply:
Should we consider adding the format to the namespacing (e.g. JSON), so when Ceramic potentially supports other schema types that it's clearly in the name? Should we consider something like `json-profile-basic` or `profile-basic-json`

Reply:
Also, as implied above, should we put the descriptor after the object type (profile-basic) so that multiple profiles will show up next to each other in the package list? It's easy to imagine something like:
- `profile-basic-json`
- `profile-basic-cson`
- `profile-dao-json`
- `profile-gitcoin-json`
- `profile-user-json`
- ...

Reply:
I don't think we need to include the format, all we support is JSON schemas anyways so it's the entire stack that would need to be updated if Ceramic were to support additional ways to create schemas.
Regarding namespacing, personally I would find it more useful as semantic relations, ex `schema.org-action`, `schema.org-creative-work`, `w3c-activitypub` so it's easier to figure out the datamodels that are more likely to be working well together, otherwise I fear we might end-up with 100s of `profile-...` that are hard to differentiate.

Reply:
Whats *CSON* 😹

Reply:
Naming is hard here. I think what Paul suggests with domain names could make sense. It's annoying because the schemas and definitions themselves have unique identifiers (streamids).

Reply:
What other semantic relations/descriptors could we imagine? I guess a lot would default to using the web3 namespace?

Reply:
Yes the downside of `web3` is that it's such a large scope, that's why I thought the company/organization name could be better suited.
Eventually, I think we could have both wide-ranging namespaces (ex `w3c-activitypub`) and more focused ones (for projects that want to share their data models but still "own" them), so our convention should be more about avoiding misleading names.

Regarding our own datamodels, maybe we can scope them to the projects we're primarily using them for? So here maybe `self.id-profile` or just `self-profile`?